webapp/src/main/java/io/github/microcks/service/MessageService.java

```diff
@@ -49,6 +51,19 @@ public class MessageService {
    private EventMessageRepository eventMessageRepository;
 
 
+   public List<? extends Exchange> computeMaxPossibleConformanceScore(Service service, Operation operation){
```

Comment:
I think the name of this method is not appropriate here... Something like `getExchangesByOperation()` would be more appropriate.